### ISSUE TITLE
Inifinite Scrolling within Branches?

### ISSUE THREAD

**Issue by httpete** [brimdata/react-arborist#213]
Hi, and thank you for your library. We have insanely large trees, and have branches that need to be loaded in "pages". Is it possible to have a branch that when opened allows us to customize the fetching to get parts (pages) of a branch and load them in an infinite scroll way? Thank you for reading.

**Comment by jameskerr** [comment]
Yes, you completely control the data that you give to react-arborist. The data fetching logic is up to you. You will have to detect when to fetch the next page. I haven't needed to do that, but it is possible.

**Comment by jameskerr** [comment]
Note to add an example with infinite scrolling...
